conda/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "eva-sub-cli" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  path: ../
+#  url: https://github.com/EBIvariation/eva-sub-cli/archive/v{{version}}.tar.gz
+#  sha256:
+
+build:
+  number: 0
+  noarch: generic
+  preserve_egg_dir: True
+#  run_exports:
+#    - {{ pin_subpackage(name, max_pin="x.x") }}
+
+requirements:
+  host:
+    - nextflow >=21.10.0
+    - python >=3.8
+    - nodejs >=10.19.1
+    - pyyaml
+    - jinja2
+    - openpyxl
+    - requests
+    - jsonschema
+  run:
+    - nextflow >=21.10.0
+    - python >=3.8
+    - pyyaml
+    - jinja2
+    - openpyxl
+    - requests
+    - jsonschema
+
+test:
+  imports:
+    - eva_sub_cli
+  commands:
+    - biovalidator --help
+    - vcf_validator --help
+    - eva-sub-cli.py --help
+
+about:
+  home: https://github.com/EBIvariation/eva-sub-cli
+  summary: EVA Submission Command Line Interface
+  license: Apache-2.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - apriltuesday
```

Comment:
Happy to add other maintainers, AFAICT the main effect is that you get tagged when the conda version is bumped (and I guess are also responsible if the build breaks!)